lib/src/git_backend.rs

```diff
@@ -664,23 +664,23 @@ fn signature_from_git(signature: gix::actor::SignatureRef) -> Signature {
     }
 }
 
-fn signature_to_git(signature: &Signature) -> gix::actor::SignatureRef<'_> {
+fn signature_to_git(signature: &Signature) -> gix::actor::Signature {
     // git does not support empty names or emails
     let name = if !signature.name.is_empty() {
-        &signature.name
+        signature.name.clone()
     } else {
-        EMPTY_STRING_PLACEHOLDER
+        EMPTY_STRING_PLACEHOLDER.to_string()
     };
     let email = if !signature.email.is_empty() {
-        &signature.email
+        signature.email.clone()
     } else {
-        EMPTY_STRING_PLACEHOLDER
+        EMPTY_STRING_PLACEHOLDER.to_string()
     };
```

Comment:
nit: these changes are unneeded since `BString` can be constructed from `&str`.

Reply:
Indeed, thanks!